pkg/agents/team1/agent1/agent.go

```diff
@@ -51,10 +51,10 @@ func (a *CustomAgent1) Run() {
 		msg := messages.NewRequestTakeFoodMessage(a.Floor(), 20)
 		a.SendMessage(1, msg)
 		a.Log("I sent a message", infra.Fields{"message": "RequestTakeFood"})
-	case 5:
-		msg := messages.NewResponseMessage(a.Floor(), true)
-		a.SendMessage(1, msg)
-		a.Log("I sent a message", infra.Fields{"message": "Response"})
+	// case 5:
+	// 	msg := messages.NewResponseMessage(a.Floor(), true, )
+	// 	a.SendMessage(1, msg)
+	// 	a.Log("I sent a message", infra.Fields{"message": "Response"})
```

Comment:
Why is this commented out?

Reply:
because responseMessage can not be sent without a requestMessage.

Reply:
Can you add in a default case handler? Otherwise this is going to error.